app/renderer/components/config_menu.js

```diff
@@ -39,22 +46,41 @@ module.exports = {
     </div>
     `,
 
-    methods: {
-        clearTasks() {
-            TaskConfig.clearTasks();
-            AppStatus.activeSuite = 0;
-        },
-        resetConfig() {
-            this.config.bottomBar=true;
-            this.config.animatedSpinner=true;
-        },
-        onClose() {
-            this.config.bottomBar = AppStatus.config.bottomBar;
-            this.config.animatedSpinner = AppStatus.config.animatedSpinner;
-        },
-        onSave() {
-            AppStatus.config.bottomBar = this.config.bottomBar;
-            AppStatus.config.animatedSpinner = this.config.animatedSpinner;
-        }
+  methods: {
+
+    exportTasks(){
+      let content = TaskConfig.getData() ;
+      content = JSON.stringify(content) ;
+
+        dialog.showSaveDialog({ filters: [
+
+          { name: 'gtasks', extensions: ['json'] }
+
+        ]}, function (fileName) {
```

Comment:
Use an arrow function here (and change fileName to filename):

(filename)=>{
....